managed/src/main/java/com/yugabyte/yw/controllers/AvailabilityZoneController.java

```diff
@@ -120,7 +93,7 @@ public Result delete(UUID customerUUID, UUID providerUUID, UUID regionUUID, UUID
       responseJson.put("success", true);
       return ApiResponse.success(responseJson);
     } catch (Exception e) {
```

Comment:
Just remove the try catch. Its not adding any new info (az code is already in request param)